#!/usr/bin/python3
def raise_exception_msg(message=""):
	"""
	Raises a NameError with the given message.
	Sometimes you just need to call someone out by name... but in error form.
	"""
	raise NameError(message)